addons/website/views/website_templates.xml

```diff
@@ -836,24 +854,24 @@
                             </small>
                         </div>
                         <div class="col-lg-4 text-lg-right">
-                            <div class="s_share no_icon_color" data-snippet="s_share" data-name="Social Media">
+                            <div class="s_share s_social_media no_icon_color" data-snippet="s_social_media" data-name="Social Media">
```

Comment:
This seems wrong, the new snippet `s_social_media` is using the `s_share` snippet scss.
It means we have a single resource (s_share/000.js) being used by 2 widgets, it will be a nightmare to maintain if we were to depreciate a snippet version one day.
Given it is only a ~50 lines file, we should duplicate it and get rid of the s_share class and replace `s_share_xxx` by `s_social_media_xxx`, don't you think? 

```suggestion
                            <div class="s_social_media no_icon_color" data-snippet="s_social_media" data-name="Social Media">
```